server.js

```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+const app = require('./lib/app');
+
+http.createServer(app)
+  .listen(7890);
```

Comment:
```suggestion
  .listen(7890, () => console.log('Server running...'));
```
Sometimes handy to have visual feedback so you know server is up and running.